cmd/eventlistener/main.go

```diff
@@ -17,19 +17,65 @@ package main
 
 import (
 	"fmt"
+	triggersAPI "github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
+	triggersClientset "github.com/tektoncd/triggers/pkg/client/clientset/versioned"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
 	"log"
 	"net/http"
+	"os"
 )
 
+func triggerBindings() ([]*triggersAPI.TriggerBinding, error) {
+	triggerBindings := make([]*triggersAPI.TriggerBinding, len(eventListener.Spec.TriggerBindingRefs))
+	for i, bindingRef := range eventListener.Spec.TriggerBindingRefs {
+		triggerBinding, err := triggersClient.TriggersV1alpha1().TriggerBindings(bindingRef.Namespace).Get(bindingRef.Name, v1.GetOptions{})
+		if err != nil {
+			return nil, fmt.Errorf("Failed to find TriggerBinding %v", bindingRef)
+		}
+		triggerBindings[i] = triggerBinding
+	}
+	return triggerBindings, nil
+}
+
 func handler(w http.ResponseWriter, r *http.Request) {
-	log.Printf("EventListener Pod received a request: %+v", r)
-	fmt.Fprintf(w, "Hello from the EventListener Pod! \n")
+	log.Printf("EventListener pod received a request: %+v", r)
+	triggerBindings, err := triggerBindings()
+	if err != nil {
+		log.Fatal(err)
+	}
+	for i := range triggerBindings {
```

Comment:
nit: we're looping through the `triggerBindings` twice here (because we already looped through the bindings in the `triggerBinding` function).

I think it would be better to get the TriggerBindings from the client here, and remove the `triggerBindings` function. So you could do something like:
```
for _, bindingRef := range eventListener.Spec.TriggerBindingRefs {
  tb, err := triggersClient.TriggersV1alpha1().TriggerBindings(bindingRef.Namespace).Get(bindingRef.Name, v1.GetOptions{})
  if err != nil {
    log.Printf("Error getting TriggerBinding %v", bindingRef)
  }
  ...
}
```